azurerm/internal/services/policy/tests/data_source_policy_definition_test.go

```diff
@@ -104,7 +152,7 @@ resource "azurerm_policy_definition" "test_policy" {
   name         = "acctestpol-%d"
```

Comment:
Could we get some uppercase letters here?

Reply:
Done